srcpkgs/eiwd/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'eiwd'
+pkgname=eiwd
+version=1.6r1
+revision=1
+wrksrc="${pkgname}-${version/r/-}"
+build_style=gnu-configure
+configure_args="--disable-systemd-service --enable-pie
+ --disable-external-ell --disable-dbus --enable-wired"
+hostmakedepends="python3-docutils pkg-config automake libtool"
+makedepends="readline-devel ell-devel>=0.31"
```

Comment:
Is the version guard necessary? I guess maybe if ell gets reverted/downgraded at some point.

Reply:
At this point `ell` is broken regardless (hence `--disable-external-ell`), this is [in anticipation of fix for 1.7](https://github.com/dylanaraps/eiwd/commit/50d398e9b74f18e436e9f8d7a89e67a0333a7377)